src/views/Main/routes.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import {Route, IndexRedirect} from 'react-router'
+import AuthService from 'utils/AuthService'
+import Container from './Container'
+import Catalog from './Catalog/Catalog'
+import Login from './Login/Login'
+
+const CONSTS = require('../../../config/consts');
```

Comment:
refer to my above comments. `../../../config/consts` is the wrong location for these Auth0 constants. they should live under `src`